lib/internal/v8_prof_polyfill.js

```diff
@@ -26,40 +26,40 @@
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 // Node polyfill
-var fs = require('fs');
-var os = {
+const fs = require('fs');
+const cp = require('child_process');
+const os = {
   system: function(name, args) {
     if (process.platform === 'linux' && name === 'nm') {
       // Filter out vdso and vsyscall entries.
-      var arg = args[args.length - 1];
+      const arg = args[args.length - 1];
       if (arg === '[vdso]' ||
           arg == '[vsyscall]' ||
           /^[0-9a-f]+-[0-9a-f]+$/.test(arg)) {
         return '';
       }
     }
-    return require('child_process').execFileSync(
-        name, args, {encoding: 'utf8'});
+    const cmd = name + ' ' + args.join(' ');
+    return cp.execSync(cmd, {encoding: 'utf8'});
```

Comment:
Hmm, that still doesn't seem to do the trick. I've attached the error I see below. Sorry I don't yet have the familiarity with bash to debug this better.

```javascript
{ [Error: spawnSync /bin/sh -c 'foo() { nm "$@" | (c++filt -p -i || cat) }; foo $@' ENOENT]
  code: 'ENOENT',
  errno: 'ENOENT',
  syscall: 'spawnSync /bin/sh -c \'foo() { nm "$@" | (c++filt -p -i || cat) }; foo $@\'',
  path: '/bin/sh -c \'foo() { nm "$@" | (c++filt -p -i || cat) }; foo $@\'',
  spawnargs: [ '-n', '-f', '/usr/lib/system/libxpc.dylib' ] }
```